src/test/compile-fail/feature-gate-abi-x86-interrupt.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Test that the x86 interrupt ABI cannot be used when abi_x86_interrupt
+// feature gate is not used.
+
+extern "x86-interrupt" fn foo() {}
+//~^ ERROR x86-interrupt ABI is experimental and subject to change
```

Comment:
In fact it [already has multiple such lines](https://github.com/rust-lang/rust/blob/6296d52ba60771232ab362c116e898aa930679ad/src/test/compile-fail/feature-gate-abi.rs#L11-L13).